Userland/Libraries/LibTLS/Certificate.h

```diff
@@ -218,7 +218,7 @@ class RelativeDistinguishedName {
     {
         auto entry = get(AttributeType::Cn);
         if (entry.has_value()) {
-            return entry.value();
```

Comment:
I am confused by this change, “entry” is still live while we were invoking the copy constructor, no?

Reply:
I was confused by this as well, and ChatGPT did _not_ help. But you are correct, this is not actually a bug in this code, but some kind of double free in the `Optional<String>`. And had I used strings with >7 characters I would have caught this in the tests... I will remove this change and fix the actual bug when I find the time.